bloom/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM coreweave/nccl-tests:2022-11-06_19-21-22.11_EST
+
+# setup python and conda
+
+RUN apt-get install -y --no-install-recommends \
+    python3 python3-dev python3-pip git libssl-dev pkg-config
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-latest-Linux-x86_64.sh
+
+ENV PATH=/opt/conda/bin:$PATH
+
+# create conda environment from file. this step installs torch.
```

Comment:
I was just curious where torch is getting installed since I do not see it in the environment.yaml file. If you can please explain how it gets installed, it LGTM.

Reply:
Torch and Transformers get installed as a dependency for the packages used in the environment, as installing the packages manually in a separate line creates a ton of package conflict errors.

Reply:
Yep, using their conda resulted in a ton of conflict errors, but it seems like leaving the torch and cudatoolkit dependencies out fixes the issue? Only downside of course is that torch version can't be explicitly specified